core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/models.ts

```diff
@@ -107,3 +108,36 @@ export interface ToolbarProps {
         hideSocialMedia: boolean;
     };
 }
+
+/**
+ * This is used for model the props of
+ * the New UVE Toolbar with Preview Mode and Future Time Machine
+ *
+ * @export
+ * @interface UVEToolbarProps
+ */
+export interface UVEToolbarProps {
+    editor: {
+        bookmarksUrl: string;
+        copyUrl: string;
+        apiUrl: string;
+    };
+    preview?: {
+        deviceSelector: {
+            apiLink: string;
+            hideSocialMedia: boolean;
+        };
```

Comment:
not sure if it will have the same props, but we can discuss that later

Reply:
Yep we can see this when implement the device selector buttons